OpenKeychain/src/main/java/org/sufficientlysecure/keychain/pgp/PgpSignatureChecker.java

```diff
@@ -134,10 +137,11 @@ boolean initializeOnePassSignature(Object dataChunk, OperationLog log, int inden
     }
 
     private void checkKeySecurity(OperationLog log, int indent) {
-        // TODO: checks on signingRing ?
```

Comment:
has the other TODO something to do with this one? 😬 
I think currently it is only checked on the signing subkey, but not if the master key is insecure.